src/divider/index.tsx

```diff
@@ -1,22 +1,25 @@
 import styled from 'styled-components'
 
 import { color, space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
 import { Divider } from './Divider'
 
 const StyledDivider = styled(Divider)`
+  /* HorizontalNormalization */
   & {
-    // Height is used instead of margins to prevent collapsing margin issues with other
-    // margin-based components.
-    height: calc(2 * ${space.m});
-    position: relative;
+    ${normalizeHorizontally}
   }
 
-  &:after {
-    position: absolute;
-    top: ${space.m};
-    content: ' ';
-    border-top: solid ${color.lightGray} 1px;
-    width: 100%;
+  & {
+    padding-top: ${space.m};
```

Comment:
I see.. using padding for normalization to avoid collapsing.. I think we should document that outter spaces are going to be used with padding to avoid margin collapse

Reply:
Added a comment